debian/kafka-streams-examples/Dockerfile

```diff
@@ -50,6 +50,8 @@ EXPOSE $KAFKA_MUSIC_APP_REST_PORT
 # base image that supports it
 ENV LANG="C.UTF-8"
 
+COPY settings.xml /root/.m2/settings.xml
```

Comment:
Just realized it will not be there for public builds, so this needs to be made conditional. Somehow.